projects/js-packages/publicize-components/src/components/panel/index.jsx

```diff
@@ -74,7 +74,7 @@ const PublicizePanel = ( { prePublish, children } ) => {
 			) }
 			{ isPostPublished && (
 				<>
-					<ShareStatusModalTrigger withWrapper analyticsData={ { location: 'editor' } } />
+					<ReSharingPanel />
```

Comment:
Should we hide this behind the feature flag so that the hooks inside that component don't run either?